File: mockez-back/src/main/java/com/mockez/service/SQLTypeService.java
package com.mockez.service;

import com.mockez.domain.model.entity.SQLType;

import java.util.List;

/**
 * @author Luong Quoc Trung, Do Quoc Viet
 */

public interface SQLTypeService {

    /**
     * SQLTYPE.01 GET /sql-type
     * @return sqlTypes
     */
    List<SQLType> getSQLTypes();
}

File: mocker-front/src/app/core/service/sql-type.service.ts
import { Injectable } from '@angular/core';
import { SqlTypeModel } from '@core/domain/model/entity/sql-type.model';
import { AbstractService } from '@core/service/abstract.service';

/**
 * @author Do Quoc Viet
 */

@Injectable({
  providedIn: 'root'
})
export class SQLTypeService extends AbstractService<SqlTypeModel> {
  override ENTITY_URL = 'sql-type';
}
